src/codeu/chat/client/commandline/Chat.java

```diff
@@ -37,6 +37,18 @@
 
   public Chat(Context context) {
     this.panels.push(createRootPanel(context));
+      panel.register(“info”, new Panel.Command() {
+        @Override
+        public void invoke(Scanner args) {
+            final ServerInfo info = context.getInfo();
+            if (info == null) {
+                System.out.println("The server did not send us a valid info object")
+                } else {
+                // Print the server info to the user in a pretty way
```

Comment:
You need to add some of your own code here that displays the uptime information to the user. Take a look at @ComarPers922 's pull request https://github.com/vgomezASC/CodeU-Summer-2017/pull/4 that adds a similar feature using the `ServerInfo` object.